Loop/Managers/RemoteDataServicesManager.swift

```diff
@@ -596,26 +596,6 @@ extension RemoteDataServicesManager {
 }
 
 extension RemoteDataServicesManager {
-
-    func serviceForPushNotification(_ notification: [String: AnyObject]) -> RemoteDataService? {
-        
-        let defaultServiceIdentifier = "NightscoutService"
-        let serviceIdentifier = notification["serviceIdentifier"] as? String ?? defaultServiceIdentifier
-        return remoteDataServices.first(where: {$0.serviceIdentifier == serviceIdentifier})
-    }
-    
```

Comment:
This method is now removed as we will not fetch a command from the plugin now.

Reply:
We are not *removing* push notifications for commands, right? Just adding periodic fetch as a backup, for when push does not work.

Reply:
Push is still the most timely way to wake Loop to handle a remote command, but it may be not 100% reliable.

Reply:
Yeah push notifications are not being removed with these updates or Remote 2.0.